deeppavlov/configs/kbqa/kbqa_rus.json

```diff
@@ -0,0 +1,262 @@
+        "load_path": "{MODELS_PATH}/ner/tag.dict",
+        "out": ["y_ind"]
+      },
+      {
+        "in": ["x_lower"],
+        "class_name": "char_splitter",
+        "out": ["x_char"]
+      },
+      {
+        "in": ["x_char"],
+        "id": "char_vocab",
+        "class_name": "simple_vocab",
+        "pad_with_zeros": true,
+        "fit_on": ["x_char"],
+        "save_path": "{MODELS_PATH}/ner/char.dict",
+        "load_path": "{MODELS_PATH}/ner/char.dict",
+        "out": ["x_char_ind"]
+      },
+      {
+        "in": ["x_san"],
+        "id": "embedder",
+        "class_name": "fasttext",
+        "pad_zero": true,
+        "load_path": "{DOWNLOADS_PATH}/embeddings/lenta_lower_100.bin",
+        "out": ["x_emb"]
+      },
+      {
+        "in": ["x_tokens"],
+        "class_name": "mask",
+        "out": ["mask"]
+      },
+      {
+        "class_name": "random_emb_mat",
+        "id": "embeddings",
+        "vocab_len": "#word_vocab.len",
+        "emb_dim": 100
+      },
+      {
+        "class_name": "random_emb_mat",
+        "id": "embeddings_char",
+        "vocab_len": "#char_vocab.len",
+        "emb_dim": 100
+      },
+      {
+        "in": ["x_emb", "mask", "x_char_ind"],
+        "in_y": ["y_ind"],
+        "out": ["y_predicted"],
+        "class_name": "ner",
+        "main": true,
+        "n_hidden_list": [128],
+        "net_type": "rnn",
+        "cell_type": "lstm",
+        "use_cudnn_rnn": true,
+        "n_tags": "#tag_vocab.len",
+        "token_emb_dim": "#embedder.dim",
+        "char_emb_dim": 100,
+        "save_path": "{MODELS_PATH}/ner/model",
+        "load_path": "{MODELS_PATH}/ner/model",
+        "char_emb_mat": "#embeddings_char.emb_mat",
+        "use_crf": true,
+        "use_batch_norm": true,
+        "embeddings_dropout": true,
+        "top_dropout": true,
+        "intra_layer_dropout": true,
+        "l2_reg": 0,
+        "learning_rate": 1e-2,
+        "dropout_keep_prob": 0.7
+      },
+      {
+        "class_name": "entity_linker",
+        "id": "linker",
+        "load_path": "{DOWNLOADS_PATH}/wikidata_rus",
+        "wiki_filename": "wiki_rus.pickle",
+        "entities_filename": "wiki_rus_name_to_q_lowercase.pickle",
+        "inverted_index_filename": "inverted_index.pickle",
+        "id_to_name_file": "wiki_rus_q_to_name.pickle",
+        "use_inverted_index": false,
+        "rule_filter_entities": true,
+        "debug": false
+      },
+      {
+        "in": ["x_tokens", "y_predicted", "y_pred_probas"],
+        "out": ["answers"],
+        "load_path": "{DOWNLOADS_PATH}/wikidata_rus/wiki_rus_q_to_name.pickle",
+        "class_name": "kb_answer_parser_wikidata",
+        "classes_vocab_keys": "#classes_vocab.keys()",
+        "top_k_classes": 5,
+        "relations_maping_filename": "wiki_properties_id_to_name.pickle",
+        "templates_filename": "templates.pickle",
+        "debug": false,
+        "return_confidences": false,
```

Comment:
Why do you need this option? It does not affect computation times nor memory usage and you can just ignore confidences afterwards